backend/infrahub/workflows/models.py

```diff
@@ -49,16 +52,20 @@ def full_name(self) -> str:
         return f"{self.name}/{self.name}"
 
     def to_deployment(self) -> dict[str, Any]:
-        payload: dict[str, Any] = {
-            "name": self.name,
-            "entrypoint": self.entrypoint,
-        }
+        payload: dict[str, Any] = {"name": self.name, "entrypoint": self.entrypoint, "tags": self.get_tags()}
         if self.type == WorkflowType.INTERNAL:
             payload["version"] = __version__
         if self.cron:
             payload["schedules"] = [DeploymentScheduleCreate(schedule=CronSchedule(cron=self.cron))]
+
         return payload
 
+    def get_tags(self) -> list[str]:
+        tags: list[str] = [WorkflowTag.WORKFLOWTYPE.render(identifier=self.type.value)]
+        for tag in self.tags:
+            tags.append(tag.render())
+        return tags
+
```

Comment:
It's a detail but `tags += [tag.render() for tag in self.tags]` would be more pythonic